projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/monitoring/DeploymentMonitor.java

```diff
@@ -81,9 +90,29 @@ public void recordDeploymentStatus(String dataJobName, DeploymentStatus deployme
   }
 
   private boolean saveDataJobStatus(
-      final String dataJobName, final DeploymentStatus deploymentStatus) {
+      final String dataJobName,
+      final DeploymentStatus deploymentStatus,
+      DataJobDeployment dataJobDeployment) {
     if (jobsRepository.updateDataJobLatestJobDeploymentStatusByName(dataJobName, deploymentStatus)
         > 0) {
+      switch (deploymentStatus) {
+          // case PLATFORM_ERROR:
```

Comment:
What is the idea of the commented out code?

Reply:
Removed it